eng/Version.Details.xml

```diff
@@ -49,13 +49,13 @@
       <Uri>https://github.com/dotnet/CliCommandLineParser</Uri>
       <Sha>0e89c2116ad28e404ba56c14d1c3f938caa25a01</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="16.9.0-preview-20502-03">
+    <Dependency Name="Microsoft.Build" Version="16.8.0-preview-20506-05">
```

Comment:
@rainersigwald @Forgind why is the version going from 16.9 back down to 16.8?

Reply:
It looks like MSBuild's master branch is still set to 16.9. I'm guessing it's because we needed real optprof data for the vs16.8 branch, and we hadn't collected it before branching for 16.9. @rainersigwald triggered a new build to collect the data; apparently that made a PR here? ([Internal link](https://teams.microsoft.com/l/message/19:3212bf033f4c4b5198643a04fa1048fa@thread.skype/1602015803800?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=4ba7372f-2799-4677-89f0-7a1aaea3706c&parentMessageId=1602015803800&teamName=.NET%20Developer%20Experience&channelName=MSBuild&createdTime=1602015803800))

Reply:
This is an indication that we've got channels set up wrong.

This one was bogus:

```sh-session
❯ darc get-subscriptions --target-repo sdk --source-repo msbuild
https://github.com/dotnet/msbuild (VS 16.8) ==> 'https://github.com/dotnet/sdk' ('master')
  - Id: 541ffb79-a58a-4b7b-321c-08d8080363b6
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: False
  - Merge Policies:
    Standard
  - Last Build: 20201002.3 (ca44138662e3aa90eb9305dd31d906ef02e962cb)
```

I just updated it to the `VS 16.9` channel and retriggered the subscription.